spec/javascript/e2e/ApplicationNewPage.e2e.js

```diff
@@ -0,0 +1,60 @@
+import puppeteer from 'puppeteer'
+
+import utils from '../support/puppeteer/utils'
+import steps from '../support/puppeteer/steps'
+import { LEASE_UP_LISTING_ID } from '../support/puppeteer/consts'
+
+describe('ApplicationNewPage', () => {
+  test('should create a new application', async () => {
+    const FIRST_NAME = 'Some first name'
+    const LAST_NAME = 'Some last name'
+    const DATE_OF_BIRTH = '03/03/1983'
+
+    let browser = await puppeteer.launch({ headless: true })
+    let page = await browser.newPage()
+
+    await steps.loginAsAgent(page)
+    await steps.goto(page, `/listings/${LEASE_UP_LISTING_ID}/applications/new`)
+
+    await page.type('#first_name', FIRST_NAME)
+    await page.type('#last_name', LAST_NAME)
+    await page.type('#date_of_birth', DATE_OF_BIRTH)
+    await page.click('.save-btn')
+    await page.waitForNavigation()
+    await steps.waitForApp(page)
+
+    const hasApplicationDetails = await utils.isPresent(page, '.application-details')
+    expect(hasApplicationDetails).toBe(true)
+
+    expect(page.url()).toMatch(/\/applications\/.*\?showAddBtn=true/)
+
+    // We get all the application attributes values
+    const values = await page.$$eval('.content-card p', elms => elms.map(e => e.textContent))
+    expect(values).toContain(FIRST_NAME)
+    expect(values).toContain(LAST_NAME)
+    expect(values).toContain(DATE_OF_BIRTH)
+
+    browser.close()
+  }, 260000)
```

Comment:
The time to wait for the test to finish. More than that the test will fail. We need to specify one to increase the default time in jest because it's not enough

Reply:
can we set it as a constant somewhere and use the constant instead of having a different one for each test?

Reply:
👍